src/Components/ModelledPropertyBuilder/ModelledPropertyBuilder.styles.ts

```diff
@@ -100,3 +102,18 @@ export const propertyExpressionLabelStyles: Partial<ILabelStyles> = {
         paddingTop: 5
     }
 };
+
```

Comment:
the vertical spacing seems off to me. Can we fix the padding to align with the "property expression" label?
![image](https://user-images.githubusercontent.com/57726991/171518863-55eaa71f-142d-4367-9f00-bb31c7769b59.png)